golint/stringsflag.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+/*
+
+This code is copied from cmd/go/build.go and should in my opinion be part of the "flag" package...
```

Comment:
hmm, yes. I'm not keen on duplicating all this. I'd prefer to only support a subset that is trivially parseable (e.g. only splitting on commas).